alacritty_macos.yml

```diff
@@ -200,5 +200,10 @@ mouse_bindings:
 
 # Shell
 #
-# You can set this to a path to your favorite shell, e.g. /bin/fish
-shell:
```

Comment:
Seems like this would install a broken config by default. Would you mind opening a separate PR for the config changes so they can be landed sooner?